src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
+
+  }
+  &__card:hover {
+    transform: scale(120%);
```

Comment:
I use numerical values to change the scaling. Not percent.
`transform: scale(1.2);`